src/app/layout.tsx

```diff
@@ -3,18 +3,21 @@ import { Metadata } from "next";
 
 import React from "react";
 import NavigationBar from "@/components/NavBar/NavigationBar";
+import { AuthRouting } from "@/app/auth";
 
 interface Props {
-    children: React.ReactElement;
+    children: React.ReactNode;
```

Comment:
Can you explain why this is `React.ReactNode` instead of `React.ReactElement`?

Reply:
React.ReactNode is the type we should be using for children. React.ReactElement means we can't use strings/numbers etc. and is causing unneccessary fragments across the codebase